packages/flutter/lib/src/material/tooltip.dart

```diff
@@ -112,9 +112,15 @@ class Tooltip extends StatefulWidget {
     this.waitDuration,
     this.showDuration,
     this.child,
+    this.triggerMode,
+    this.enableFeedback,
   }) : assert(message != null),
        super(key: key);
 
+  /// The default value used when a value is not provided anywhere.
+  /// Read documentation for specific properties to determine all sources.
```

Comment:
```suggestion
  ///
  /// Read documentation for specific properties to determine all sources.
```
Read what documentation?

Reply:
We tend to put default constants in _k values at the top of files. The CupertinoScrollbar has a good example of this.